src/Tizen.NUI/src/public/Utility/GraphicsTypeExtensions.cs

```diff
@@ -1,4 +1,21 @@
-﻿using System.ComponentModel;
+﻿/*
+ * Copyright(c) 2021 Samsung Electronics Co., Ltd.
```

Comment:
Minor comment : Could you update the copyright year? Welcome to 2022 ;)

Reply:
this cs file was generated at 21 so I left it 21.
other file which created new was named 22 :)